README.md

```diff
@@ -36,6 +36,8 @@
 
 This is an experimental web client for Jellyfin based on Vue.js. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start.
 
+An always up-to-date build from master branch is available at https://jellyfin-vue.pages.dev/
```

Comment:
We could maybe use [this](https://img.shields.io/badge/pages.dev-master-success) instead?

I'm mainly not quite sure about the formulation of this and the place it should live in inside the readme.

Reply:
I'm fine with either but I think the direct mention is *slightly* better for users that don't know how to use GitHub and are not used to understand/see those badges.

Most projects have always a "View Demo" or "Online Demo" mention in their readmes rightaway, no badges in the middle. 

Perhaps I should change the formulation to "Demo" as well.

Reply:
Yeah, I think a "View demo" link would fit better :)

Reply:
I agree with @ferferga here, the badges aren't very good for newcomers, an explicit link would probably be best.

Reply:
I agree too, but a big "View demo" link feels better, then, as proposed by ferferga :yum: